flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/utils/ExecNodeUtil.java

```diff
@@ -35,6 +38,11 @@
 /** An Utility class that helps translating {@link ExecNode} to {@link Transformation}. */
 public class ExecNodeUtil {
 
+    /** Return bytes size for given option in {@link TableConfig}. */
+    public static long getMemorySize(TableConfig tableConfig, ConfigOption<String> option) {
+        return MemorySize.parse(tableConfig.getConfiguration().getString(option)).getBytes();
```

Comment:
ConfigOption supports type of MemorySize, can we just use it?,ref:FRAMEWORK_HEAP_MEMORY

Reply:
It will broken current API, because the compile error will occur when user's code is `configuration.set(
      ExecutionConfigOptions.TABLE_EXEC_RESOURCE_HASH_AGG_MEMORY, "128 mb")`

Reply:
I create an issue to do the improvement, see https://issues.apache.org/jira/browse/FLINK-20879